src/core/qgis.cpp

```diff
@@ -46,6 +46,48 @@ const int Qgis::QGIS_VERSION_INT = VERSION_INT;
 
 // Release name
 const QString Qgis::QGIS_RELEASE_NAME( QStringLiteral( RELEASE_NAME ) );
+// Gdal Version string at build/compile time
+QString Qgis::GDAL_BUILD_VERSION( QString::fromUtf8( GDAL_RELEASE_NAME ) );
+const int Qgis::GDAL_BUILD_VERSION_MAJOR = GDAL_VERSION_MAJOR;
+const int Qgis::GDAL_BUILD_VERSION_MINOR = GDAL_VERSION_MINOR;
+const int Qgis::GDAL_BUILD_VERSION_REV = GDAL_VERSION_REV;
+QString Qgis::GDAL_RUNTIME_VERSION = QString::null;
```

Comment:
Yes - you'll also need to update the check on line 62 to use .isEmpty() instead. As per the QString docs - "We recommend that you always use the isEmpty() function and avoid isNull().". My suspicion is that Qt will remove support for QString::null in a future release, so better for us to follow that recommendation now and avoid work in future.